content/en/docs/ambient/usage/troubleshoot-waypoint/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Troubleshoot connectivity issues with waypoints
```

Comment:
Could be waypoint not enforcing policies also... that is when user would mostly need this doc for. why their waypoint isn't working, or why traffic is broken.

@ilrudie @craigbox thoughts?